src/Network/Network/VirtualNetworkGateway/NewAzureVpnClientConfigurationCommand.cs

```diff
@@ -55,7 +55,7 @@ public class NewAzureVpnClientConfigurationCommand : VirtualNetworkGatewayBaseCm
         public string ProcessorArchitecture { get; set; }
 
         [Parameter(
-            Mandatory = true,
+            Mandatory = false,
```

Comment:
Need to regenerate help doc for this cmdlet

Reply:
I see. Sure.